src/app/api/user/route.ts

```diff
@@ -0,0 +1,30 @@
+import { NextResponse, NextRequest } from "next/server";
+import connectDatabase from "@/lib/database";
+import UserSignee from "@/models/user-model";
+
+export async function POST(request: NextRequest) {
+
+    const { signeeWalletAddress } = await request.json();
+
+    await connectDatabase();
```

Comment:
Normally we need to put this in a global place, not require to be in every route.
At least in node js

Reply:
I will find a solution to this comment in next PR, this is a method related to next js 13 + versions